.github/workflows/qiita-ci.yml

```diff
@@ -7,11 +7,12 @@ on:
 jobs:
   # derived from https://github.com/actions/example-services/blob/master/.github/workflows/postgres-service.yml
   main:
+    # 7/16/24: confirm current ubuntu-latest is still 22.04.
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
-        python-version: ["2.7", "3.9"]
+        python-version: ["3.9"]
```

Comment:
Can you bring back 3.11?

Reply:
For reference please see our comments @ https://github.com/qiita-spots/qiita_client/pull/52 . We will be bringing support to 3.11 when we migrate to pytests or perhaps nose3.